### ISSUE TITLE
generate an intro GET / on lux new

### ISSUE THREAD

**Issue by kevboh** [postlight/lux#331]
Let me know if you think is a good idea and I'll open a PR. After `lux new` this is what `localhost:4000` returns:

```
{"errors":[{"status":"404","title":"Not Found"}],"jsonapi":{"version":"1.0"}}
```

On Safari it just downloads a JSON blob 😢 . Maybe instead it could show a good next step, like a CLI command to generate a new controller?

**Comment by javamonn** [comment]
This sounds like a good idea to me, the old [rails hello world page](http://guides.rubyonrails.org/v4.2/images/getting_started/rails_welcome.png) is good inspiration - maybe some links back to documentation guides as well?

**Comment by zacharygolba** [comment]
This is a great idea @kevboh!

**Comment by kevboh** [comment]
Okay, cool. I am Bad At HTML but can give it a simple shot and maybe ask a designer for help!

**Comment by gentle-noah** [comment]
I would love to pick this up if you can give me more info on [this](https://github.com/postlight/lux/issues/51)

**Comment by zacharygolba** [comment]
@gentle-noah I appreciate the enthusiasm!

Currently, we a working on a landing page/documentation website for Lux. I think we should hold off on this issue until we finish the landing page for the sake of normalizing the styles.

I'll keep you posted about this issue when more information is available. 😃

**Comment by gentle-noah** [comment]
@zacharygolba - right on. Let me know if you need a hand with any of that. Some recent things I have built that called for a lot of HTML and CSS would be [Zooppa Marketing](www.zooppa.com) and [Zooppa Community](www.community.zooppa.com/en-us)

**Comment by nickschot** [comment]
ember-welcome-page is another fine example of this. See: https://github.com/ember-cli/ember-welcome-page and more specifically: https://github.com/ember-cli/ember-welcome-page/blob/master/addon/templates/components/welcome-page.hbs

I think some similar pointers to docs/gitter etc. would be great.

**Comment by CzechsMix** [comment]
Would that absolve the need to map an action to "GET /" ?

**Comment by nickschot** [comment]
No, that's still needed if we want it to be visible on '/' (which we want).
